lib/archethic/shared_secrets/node_renewal.ex

```diff
@@ -150,7 +148,7 @@ defmodule ArchEthic.SharedSecrets.NodeRenewal do
     Transaction.new(
       :origin_shared_secrets,
       %TransactionData{
-        content: new_origin_shared_pub_keys,
+        content: <<origin_public_keys_bin <> new_origin_shared_pub_keys>>,
```

Comment:
I don't think we need the origin public keys here, as they will be in the `ownerships` as authorized keys